src/Redux/ConfigureStore.js

```diff
@@ -1,11 +1,14 @@
 import { createStore, combineReducers, applyMiddleware } from 'redux';
+import thunk from 'redux-thunk';
```

Comment:
- [ ] I was unable to successfully run your app because you have a missing dependency in your app. When using npm packages, kindly ensure they are installed in your `package.json` file.

  You can run `npm i redux-thunk` to resolve this issue.

  ![ayoub-thunk](https://user-images.githubusercontent.com/36542986/160427163-a4543982-55ad-4a87-b57e-109fb2590d25.jpg)

Reply:
Done